addons/dex-k8s-authenticator/1.1.x/dex-k8s-authenticator-9.yaml

```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: dex-k8s-authenticator
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: dex-k8s-authenticator
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.2.0-1"
+    appversion.kubeaddons.mesosphere.io/dex-k8s-authenticator: "v1.2.0"
```

Comment:
I think we need to put this addon revision in a new `1.2.x` directory that reflects this minor version bump, since it's currently in `addons/dex-k8s-authenticator/1.1.x/`.